cmd/check.go

```diff
@@ -21,6 +21,14 @@ var checkFileCmd = &cobra.Command{
 		path := args[1]
 
 		runtimeData := getInitialRuntimeData(cmd)
+		defer func() {
```

Comment:
In go there is no way of implementing destructor on struct so I created method `runtimeData.DeleteRuntimeTemp()` which responsibility is to delete temporary files. I putted it into defer in order to execute it at the end of that cmd function.
If you ask why it is invoked inside anonymous function - it is because it is the only way to potentially handle error.